src/Roslyn.Suppressions/Roslyn.Suppressions/Settings.Cache/ISettingsCache.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SonarQube.Client.Models;
+
+namespace SonarLint.VisualStudio.Roslyn.Suppressions.Settings.Cache
+{
+    public interface ISettingsCache
+    {
+        IEnumerable<SonarQubeIssue> GetSettings(string projectKey);
```

Comment:
We're calling the key `settingsKey` rather than `projectKey` since it isn't really the project key.